### PR TITLE
Add local host and forward address for container domain

### PR DESCRIPTION
**Pull request by ygoyal18**
Mirroring #301. @OPpuolitaival this is to investigate why PR checks are not catching the problem. Once we have that I can update this PR with the fix. 

1. Modify edge-proxy config to add new forwarding address for containers domain
1. Add `containers.local` to the list of known hosts

### CODE REVIEW

File: recipes-wigwag/edge-proxy/files/launch-edge-proxy.sh [hunk 1]
```diff
@@ -48,4 +49,4 @@ exec /wigwag/system/bin/edge-proxy \
 -cert-strategy-options=path=/1/pt \
 -cert-strategy-options=device-cert-name=mbed.LwM2MDeviceCert \
 -cert-strategy-options=private-key-name=mbed.LwM2MDevicePrivateKey \
--forwarding-addresses={\"gateways.local\":\"${GATEWAYS_ADDRESS#"https://"}\"}
+-forwarding-addresses={\"gateways.local\":\"${GATEWAYS_ADDRESS#"https://"}\",\"containers.local\":\"${CONTAINERS_ADDRESS#"https://"}\"}
```
Comment (ashishwigwag) [inline recipes-wigwag/edge-proxy/files/launch-edge-proxy.sh:52]:
```suggestion
-forwarding-addresses={\"gateways.local\":\"${GATEWAYS_ADDRESS#"https://"}\"\,\"containers.local\":\"${CONTAINERS_ADDRESS#"https://"}\"}
```

**Review by ashishwigwag — COMMENTED** [review]
I missed reading this is to mirror the other PR.
Some suggestions to be applied later.

**Comment by esajaa** [comment]
I tried looking into this a bit today. I couldn't really find the root cause for the pass yet

File: recipes-wigwag/edge-proxy/files/launch-edge-proxy.sh [hunk 2]
```diff
@@ -21,9 +21,14 @@ EDGE_PROXY_DEBUG=false
...
 EDGE_K8S_ADDRESS=$(jq -r .edgek8sServicesAddress /userdata/edge_gw_config/identity.json)
 GATEWAYS_ADDRESS=$(jq -r .gatewayServicesAddress /userdata/edge_gw_config/identity.json)
+CONTAINERS_ADDRESS=$(jq -r .containerServicesAddress /userdata/edge_gw_config/identity.json)
 DEVICE_ID=$(jq -r .deviceID /userdata/edge_gw_config/identity.json)
 EDGE_PROXY_URI_RELATIVE_PATH=$(jq -r .edge_proxy_uri_relative_path /wigwag/etc/edge-proxy.conf.json)
 
+      if ! grep -q "containers.local" /etc/hosts; then
+          echo "127.0.0.1 containers.local" >> /etc/hosts
+      fi
+
```
Comment (ygoyal18) [inline recipes-wigwag/edge-proxy/files/launch-edge-proxy.sh:31]:
```suggestion
if ! grep -q "containers.local" /etc/hosts; then
    echo "127.0.0.1 containers.local" >> /etc/hosts
fi

```

**Pull request merged**